test/common/index.js

```diff
@@ -102,6 +102,9 @@ if (process.argv.length === 2 &&
 
 const isWindows = process.platform === 'win32';
 const isAIX = process.platform === 'aix';
+// On IBM i, process.platform and os.platform() both return 'aix', 
+// It is not enough to differentiate between IBM i and real AIX system.
+const isIBMi = execSync(`uname -s`).toString() === 'OS400\n';
```

Comment:
is `uname` guaranteed to exist on all platforms?

Reply:
```suggestion
const isIBMi = isAIX && execSync(`uname -s`).toString() === 'OS400\n';
```

This should be done for performance reasons alone, we shouldn’t just spawn a child process every time we run a test.

Reply:
It still spawns a child for every test on AIX. I'd make it a function.

Reply:
Can this use `os.type()` instead of running `uname`? That would eliminate the need to make it a function instead of a property. But I'm not sure what `os.type()` returns on IBM i.